packages/fxa-payments-server/src/routes/Product.tsx

```diff
@@ -49,6 +45,11 @@ export const Product = ({
   resetCreateSubscription,
   fetchPlansAndSubscriptions,
 }: ProductProps) => {
+  const {
+    accessToken,
+    queryParams,
```

Comment:
I would expect the actual values to be in the context (in this case `queryParams.plan` instead of `queryParams`)